datafusion/physical-plan/src/joins/join_hash_map.rs

```diff
@@ -168,6 +168,9 @@ pub trait JoinHashMapType {
     /// Returns a reference to the next.
     fn get_list(&self) -> &Self::NextType;
 
+    // Whether values in the hashmap are distinct (no duplicate keys)
+    fn is_distinct(&self) -> bool;
```

Comment:
```suggestion
    fn is_distinct(&self) -> bool {
        false
    }
```

I don't know if there are other implementers of this trait, but it is technically `pub`...